pages/common/figlet.md

```diff
@@ -23,3 +23,11 @@
 - Show available FIGlet fonts:
 
 `showfigfonts {{optional_string_to_display}}`
+
+- Use the full width of your [t]erminal and [c]enter the inputting text:
```

Comment:
```suggestion
- Use the full width of the [t]erminal and [c]enter the input text:
```

Reply:
Sensible choice, I used "inputting" word because it was already used in the first example command so I'm going to change that also